ansible-fmw/wls_variables.yml

```diff
@@ -15,7 +15,7 @@ JDK_DIR: '{{ JAVA_DIR }}/jdk1.8.0_331'
 JAVA_HOME: '{{ JAVA_DIR }}/java_latest'
 
 ## Oracle ENV Variables
-ORACLE_BASE: "/home/dojowls/wls_lite"
+ORACLE_BASE: "/home/{{ WLS_USER }}/fmw"
```

Comment:
`fmw` is general name of all Fusion Middleware which covers many :)  
this playbook is _only_  about WLS , therefore `wls-server` or even `wls-lite` is better choice.

Reply:
The changes have been done. Please check